internal/test/go.mod

```diff
@@ -18,3 +18,5 @@ require (
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v0.18.2
 )
+
+replace github.com/openshift/windows-machine-config-bootstrapper/tools/windows-node-installer => ../../tools/windows-node-installer
```

Comment:
IIRC, we wanted to pin to the old version of WNI. However it doesn't matter as long as the tests pass for now as @PratikMahajan  is working on removing it.

Reply:
Lmk if I should drop this commit :)

Reply:
To be on the safe side, yes.

Reply:
done